src/client/Microsoft.Identity.Client/Platforms/iOS/Broker/iOSBroker.cs

```diff
@@ -377,7 +377,7 @@ public static void SetBrokerResponse(NSUrl responseUrl)
         /// </summary>
         public Task RemoveAccountAsync(ApplicationConfiguration applicationConfiguration, IAccount account)
         {
-            throw new NotImplementedException();
```

Comment:
ClientApplicationBase.RemoveAsync has also been changed. It used to do nothing because we were checking silent auth.. .and now it also does nothing.

https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/2825/files#diff-69d983917369bc65e1b11ec08e7e8a8dc7a45a2d7b32902fb82da7d5a2989dceR204